### PR TITLE
[imporve] null judge modify.

### PR DESCRIPTION
**Pull request by MRgenial**
## Checklist
null judge modify.

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [x]  I have written the necessary doc or comment.
- [x]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [x] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

File: manager/src/main/java/org/apache/hertzbeat/manager/service/impl/MonitorServiceImpl.java [hunk 1]
```diff
@@ -214,7 +214,7 @@ public void addMonitor(Monitor monitor, List<Param> params, String collector) th
         }).collect(Collectors.toList());
         appDefine.setConfigmap(configmaps);
 
-        long jobId = collector == null ? collectJobScheduling.addAsyncCollectJob(appDefine, null) :
+        long jobId = !StringUtils.hasText(collector) ? collectJobScheduling.addAsyncCollectJob(appDefine, null) :
```
Comment (yuluo-yx) [inline manager/src/main/java/org/apache/hertzbeat/manager/service/impl/MonitorServiceImpl.java:217]: maybe use `isNotBlank()/isBlank`?

**Review by yuluo-yx — DISMISSED** [review]
LGTM, thanks @MRgenial

File: manager/src/main/java/org/apache/hertzbeat/manager/service/impl/MonitorServiceImpl.java (hunk 1 above)
Comment (MRgenial) [reply manager/src/main/java/org/apache/hertzbeat/manager/service/impl/MonitorServiceImpl.java]: > maybe use `isNotBlank()/isBlank`?

Hi, these have been replaced.

**Review by yuluo-yx — APPROVED** [review]
LGTM.

**Comment by yuluo-yx** [comment]
> LGTM.

@Calvin979 @tomsun28 pls ptal, if no other question, I will merge to main branch.

**Review by Calvin979 — APPROVED** [review]

**Review by tomsun28 — APPROVED** [review]
check passed. 👍 It is recommended that we can deploy and check the modified functions again by using nightly docker image https://hub.docker.com/r/apache/hertzbeat/tags

**Pull request merged**